03_Modules/Ocpi/src/modules/temp/exceptions/not.found.exception.ts

```diff
@@ -0,0 +1,6 @@
+export class NotFoundException extends Error {
```

Comment:
added some exceptions, and the `UnauthorizedException` had to be added to `Base` - the exceptions are handled by the `GlobalExceptionHandler` to return the appropriate response